feature-utils/poly-look/src/react-components/icons/chevron-down.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export function ChevronDown({ fill, ...rest }) {
+  return (
+    <svg
+      data-testid="chevron-down"
+      {...rest}
```

Comment:
This is interesting... So fill and some attributes are parametrized.

Reply:
I was interested only in the `fill` attribute, but I've added `rest` for flexibility, since I'm not sure yet how this components will look like.